test/functional/terminal/buffer_spec.lua

```diff
@@ -258,6 +258,13 @@ describe(':terminal buffer', function()
   it('handles wqall', function()
     eq('Vim(wqall):E948: Job still running', exc_exec('wqall'))
   end)
+
+  it('does not segfault when pasting empty buffer #13955', function()
+    feed_command('terminal')
+    feed('<c-\\><c-n>')
+    feed_command('put a') -- buffer a is empty
+    eq(2, eval('1+1')) -- check nvim still running
```

Comment:
I think we have something like `helper.alive()` to exactly check this kind of things.

Reply:
Updated here and the other 2 tests as well in this file.